MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst LICENSE NOTICE HISTORY.rst requests/cacert.pem
+recursive-include tests *.py
```

Comment:
Perhaps `graft tests` would be a more permissive command to use here. That way we're capturing possibly required files in the tests directory that aren't necessarily python files.